stackit/internal/services/rabbitmq/credential/resource.go

```diff
@@ -37,7 +37,6 @@ type Model struct {
 	Host         types.String `tfsdk:"host"`
 	Hosts        types.List   `tfsdk:"hosts"`
 	HttpAPIURI   types.String `tfsdk:"http_api_uri"`
-	Name         types.String `tfsdk:"name"`
```

Comment:
RabbitMQ credentials has new fields:
- `HttpApiUris`, list of strings (similar implementation as `hosts`)
- `Management`, string
- `Uris`, list of strings (similar implementation as `hosts`)

Can you add them?

Reply:
added all new fields